sdk/policy/arm-policy/CHANGELOG.md

```diff
@@ -154,9 +105,19 @@ Compared with version 5.1.0
   - Interface PolicySetDefinitionsListBuiltInOptionalParams has a new optional parameter expand
   - Interface PolicySetDefinitionsListByManagementGroupOptionalParams has a new optional parameter expand
   - Interface PolicySetDefinitionsListOptionalParams has a new optional parameter expand
-  - Added Enum KnownAssignmentScopeValidation
+  - Added Enum KnownAssignmentType
   - Added Enum KnownOverrideKind
   - Added Enum KnownSelectorKind
+
+### Breaking Changes
+
+  - Removed operation group DataPolicyManifests
```

Comment:
Please ensure these breakings are expected.

Reply:
![image](https://github.com/user-attachments/assets/577f1663-572c-406e-97d1-b97d1901ef67)

Comfirmed with the service to accept this breaking.